### ISSUE TITLE
Add missing alt tags and aria-label to links and images

### ISSUE THREAD

**Issue by julianguyen** [ifmeorg/ifme#973]
# Description 

Add missing alt tags and aria-label to links and images

Helpful: https://a11yproject.com/

---

Please assign yourself (via the Assignees dropdown), if you do want to work on this issue. Can't find yourself? You need to [join our organization](https://github.com/ifmeorg/ifme/wiki/Join-Our-Slack).

Check out our [Picking Up Issues](https://github.com/ifmeorg/ifme/wiki/Picking-Up-Issues) guide if you haven't already!

**Comment by devkant** [comment]
I would like to work on it

**Comment by julianguyen** [comment]
Hey @devkant, we would love for you to work on this! Here are next steps for joining: https://github.com/ifmeorg/ifme/wiki/Join-Our-Slack

**Comment by digilou** [comment]
Hi @devkant are you working on this? If not, @julianguyen I'd like to take it on.

**Comment by madhubs** [comment]
If any additional help is needed for this issue, I would love to contribute in any way, but understand it's been assigned to @digilou.

**Comment by digilou** [comment]
@madhubs You can have this issue! I'll unassign myself.

**Comment by madhubs** [comment]
> @madhubs You can have this issue! I'll unassign myself.

Thank you @digilou!

**Comment by RossellaFer** [comment]
Hi, if you need an extra set of hands on this, I am available!

**Comment by julianguyen** [comment]
Hey @RossellaFer and @taramatt93, we'd love your help on this! Please follow [next steps](https://github.com/ifmeorg/ifme/wiki/Join-Our-Slack) for joining our contributor community.

**Comment by hnarasaki** [comment]
I would like to try this one.

**Comment by madhubs** [comment]
Yes do it!

--
Madeline Stevens
Full Stack JavaScript Developer
www.madelinerstevens.com
madelinerstevens@me.com
206-498-4786

**Comment by masenmatthews** [comment]
Hey, is anyone working on this? I would love to take it on if possible.

**Comment by hnarasaki** [comment]
@masenmatthews Please feel free to take this one! :)

**Comment by masenmatthews** [comment]
Perfect, I will aim to have it done within the next 1-2 weeks 👍

**Comment by madhubs** [comment]
Do it! Let me know what you find. It looked like a lot of labels had maybe been added already. 

--
Madeline Stevens
Trilogy Education TA
Full Stack JavaScript Developer
www.madelinerstevens.com
madelinerstevens@me.com
206-498-4786

**Comment by hnarasaki** [comment]
If this issue is still open, I'd like to try it at the Write/Speak/Code event tonight! :)

**Comment by madhubs** [comment]
Yes! It’s all yours!!!

-Maddy

**Comment by Ceheiss** [comment]
Hi, is any help still needed? This would be my first time contributing to open source.
Best regards,

**Comment by DivyaGupta261** [comment]
@hnarasaki @Ceheiss Can I take this up?

**Comment by julianguyen** [comment]
@hnarasaki are you still working on this? If not, it looks like @Ceheiss can pick it up!

**Comment by dansoud** [comment]
Could I work on this?

**Comment by hnarasaki** [comment]
@Ceheiss @julianguyen Apologies for not responding earlier - @Ceheiss , please feel free to! Good luck and have fun! :D

**Comment by Ceheiss** [comment]
great @hnarasaki thanks!

**Comment by janessatran** [comment]
Hey @dansoud! I would also like to work on this, do you think we could split it up and I could do the images and you do the links? :)  If you want to do it all, that's fine too!! Thanks in advance!

**Comment by anushkajj** [comment]
Hi! Beginner here. Is anyone working on this issue? Can I take it?
Thanks in advance!

**Comment by dikshagoyal26** [comment]
I would like to work on it

**Comment by Ceheiss** [comment]
Hi everyone, I had some personal issues so couldn't do it. I'm sorry about this, please feel free to take it

**Comment by julianguyen** [comment]
Hi @dikshagoyal26, you can assign yourself to the issue once you [join](https://github.com/ifmeorg/ifme/wiki/Join-Our-Slack) our contributor community!

**Comment by C3S4R780** [comment]
is the issue still open? I would like to work on it!

**Comment by stefanosu** [comment]
Hi is this issue still available?

**Comment by lilivelazquezz** [comment]
Hello! Is this issue still available?

**Comment by julianguyen** [comment]
Thanks for showing interest @C3S4R780 @stefanosu @lilivelazquezz! Always great to have a lot of interest in a11y.  

You could all work on it together!

Check out the footer in the issue description above for more info on picking up issues.

**Comment by drowvoloper** [comment]
@C3S4R780 @stefanosu @lilivelazquezz are you finally working on this?

**Comment by C3S4R780** [comment]
> 
> 
> @C3S4R780 @stefanosu @lilivelazquezz are you finally working on this?

I'm really sorry, i dont know much about Ruby!

**Comment by stefanosu** [comment]
@drowvoloper yes I'm working on this

**Comment by drowvoloper** [comment]
> > @C3S4R780 @stefanosu @lilivelazquezz are you finally working on this?
> 
> I'm really sorry, i dont know much about Ruby!

Oh, don't worry! I just wanna help with this issue if I can be useful. 

P.S.: Don't let Ruby frighten you!

**Comment by prasanth1308** [comment]
I would like to work on it

**Comment by paulomonteiro1** [comment]
Is anybody working on this issue? I would like to help.

**Comment by ribeirolm** [comment]
Is this issue still up for grabs? If so, I'd like to contribute :)

**Comment by afhare** [comment]
If this issue is up for grabs, or if it could be a tag-team approach, I'd love to contribute!

**Comment by oliviadavis593** [comment]
Is this something I can work on? I followed the next steps already. Don't know if I can assign myself

**Comment by hassan11196** [comment]
help needed?

**Comment by julianguyen** [comment]
Looks like it's currently not being worked on so feel free to assign yourselves @oliviadavis593  and @hassan11196! You two could even work together on it :)

**Comment by Manasa2850** [comment]
Can I work on this?

**Comment by GayathriVenkatesh** [comment]
@julianguyen Is anyone currently working on this? I would like to take it up.

**Comment by julianguyen** [comment]
@Manasa2850 @GayathriVenkatesh Feel free to assign yourselves to it! Yeah, looks like no one is working on it.

**Comment by GayathriVenkatesh** [comment]
@julianguyen There are many blank images like this one in the README files:
https://opencollective.com/ifme/sponsor/6/avatar.svg
What do you suggest I do about them?

**Comment by julianguyen** [comment]
@GayathriVenkatesh Good question and idea to also add alt tags to our README files! So this is a tricky one since those Open Collective SVG links are auto-generated.

The best course of action is to wrap the entire section of links in a `<section role="presentation">` so that screen-readers skip this content.

**Comment by govardhan27** [comment]
Hey @julianguyen , Can I take this up?

**Comment by GayathriVenkatesh** [comment]
Hi @govardhan27, I am currently working on this issue and have almost completed it. Sorry for the delay!

**Comment by discombobulateme** [comment]
Dear community, is this issue still open? If so, may I work on it? :)
